Makefile

```diff
@@ -227,104 +236,103 @@ DELIM=::
 endif
 
 
-dockerrun-p2p:
+dockerrun-p2p-elk:
```

Comment:
How important is it to rename this and all the other make recipes to add `-elk`? I think you could just modify the recipes to use `dockerbuild-test-elk` without renaming them all no?

Reply:
tests before were named without the elk and because I had to change them and keep the old ones I added the "-elk"
I don't think it's something I need to change, if anyone has a strong objection I can consider it.